src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs

```diff
@@ -62,7 +62,7 @@ public override void Render(LogEvent logEvent, TextWriter output)
 
                 if (_token.Alignment.HasValue)
                 {
-                    var str = writer.ToString();
+                    var str = writer.ToString() ?? throw new InvalidOperationException("The output TextWriter, return a invalid state.");
```

Comment:
`StringWriter.ToString()` should never return `null` (in this case the target will always be of that type); perhaps `!` non-null assertion would be suitable, here?